mitxpro/utils_test.py

```diff
@@ -117,3 +119,19 @@ def test_first_or_none():
     assert first_or_none(set()) is None
     assert first_or_none([1, 2, 3]) == 1
     assert first_or_none(range(1, 5)) == 1
+
+
+def test_unique():
+    """
+    Assert that unique() returns a generator of unique elements from a provided iterable
+    """
+    assert set(unique([1, 2, 2, 3, 3, 0, 3])) == {0, 1, 2, 3}
```

Comment:
Is `unique` meant to leave items in the order they first appeared in the list? If so we should do something like `assert list(unique(...)) == [1, 2, 3, 0]`.